src/worldcereal/job.py

```diff
@@ -100,80 +102,65 @@ def generate_map(
 
     Raises:
         ValueError: if the product is not supported
+        ValueError: if the out_format is not supported
```

Comment:
Needs to be discussed but in the preprocessing.py file we are using a different docstyle (numpy)

Ideally we should choose a unique doc style for the whole project. GFMAP is also using the numpy docstyle

For example:
https://github.com/WorldCereal/worldcereal-classification/blob/main/src/worldcereal/openeo/preprocessing.py#L35

Reply:
ah good point, I just use the autodocstring extension in vscode which produces this stuff. Should check if I can configure that somehow.

Reply:
Apparently it was indeed a setting in the extension. Updated the docstrings to numpy style.